GitUI/CommandsDialogs/BrowseDialog/DashboardControl/Dashboard.cs

```diff
@@ -16,20 +16,20 @@ namespace GitUI.CommandsDialogs.BrowseDialog.DashboardControl
 {
     public partial class Dashboard : GitModuleControl
     {
-        private readonly TranslationString cloneFork = new TranslationString("Clone {0} repository");
-        private readonly TranslationString cloneRepository = new TranslationString("Clone repository");
-        private readonly TranslationString cloneSvnRepository = new TranslationString("Clone SVN repository");
-        private readonly TranslationString createRepository = new TranslationString("Create new repository");
-        private readonly TranslationString develop = new TranslationString("Develop");
-        private readonly TranslationString donate = new TranslationString("Donate");
-        private readonly TranslationString issues = new TranslationString("Issues");
-        private readonly TranslationString openRepository = new TranslationString("Open repository");
-        private readonly TranslationString translate = new TranslationString("Translate");
-        private readonly TranslationString directoryIsNotAValidRepositoryCaption = new TranslationString("Open");
-        private readonly TranslationString directoryIsNotAValidRepository = new TranslationString("The selected item is not a valid git repository.\n\nDo you want to abort and remove it from the recent repositories list?");
-        private readonly TranslationString directoryIsNotAValidRepositoryOpenIt = new TranslationString("The selected item is not a valid git repository.\n\nDo you want to open it?");
+        private readonly TranslationString _cloneFork = new TranslationString("Clone {0} repository");
+        private readonly TranslationString _cloneRepository = new TranslationString("Clone repository");
+        private readonly TranslationString _cloneSvnRepository = new TranslationString("Clone SVN repository");
+        private readonly TranslationString _createRepository = new TranslationString("Create new repository");
+        private readonly TranslationString _develop = new TranslationString("Develop");
+        private readonly TranslationString _donate = new TranslationString("Donate");
+        private readonly TranslationString _issues = new TranslationString("Issues");
+        private readonly TranslationString _openRepository = new TranslationString("Open repository");
+        private readonly TranslationString _translate = new TranslationString("Translate");
+        private readonly TranslationString _directoryIsNotAValidRepositoryCaption = new TranslationString("Open");
+        private readonly TranslationString _directoryIsNotAValidRepository = new TranslationString("The selected item is not a valid git repository.\n\nDo you want to abort and remove it from the recent repositories list?");
+        private readonly TranslationString _directoryIsNotAValidRepositoryOpenIt = new TranslationString("The selected item is not a valid git repository.\n\nDo you want to open it?");
```

Comment:
⚠️  I've just realised we can't do this to `TranslationString` fields without causing massive headache for our translators.
Each of these fields has a corresponding mapping in *.xlf files. If we rename fields, we must rename the corresponding entries in xlf files. Otherwise translators will have to translate the lot again. This would be a major undertaking...

That's how I understand the process. 
@jbialobr / @KindDragon: am I correct?

Reply:
Yes, you right in general but:
1) We ignore underscore for TranslationString identificators
2) Transifex support Translation Memory Fill-up (Checking this option will automatically translate phrases with exact matches from the Translation Memory.)

Reply:
Phewww... we're in a good shape then. False alarm.
Thank you!